pkgs/top-level/python-aliases.nix

```diff
@@ -513,6 +513,7 @@ mapAliases ({
   recursivePthLoader = recursive-pth-loader; # added 2024-01-07
   rednose = throw "rednose is no longer maintained (since February 2018)"; # added 2023-08-06
   repeated_test = repeated-test; # added 2022-11-15
+  reportengine = throw "reportengine has been removed, since it is unmaintained and broken"; # added 2024-08-21
```

Comment:
Please name it explicitly

Reply:
What do you mean?

Reply:
I am not sure to see what is problematic here